test/system/projects/samples_test.rb

```diff
@@ -194,7 +194,7 @@ class SamplesTest < ApplicationSystemTestCase
       assert_no_selector 'table#samples-table tbody tr', text: @sample1.name
```

Comment:
Can you please add some testing for when a metadata template is selected and assert that the headers/values are in the table?

Reply:
I want to leave this for when the actual service is working and we get the templates stored instead of hard-coded into the controller.  That way the test can be set up correctly.

Reply:
I guess that's acceptable...for now :)